pkg/daemon/sync.go

```diff
@@ -70,6 +70,8 @@ func (d *Daemon) Sync(ctx context.Context, started time.Time, newRevision string
 		return err
 	}
 
+	d.Logger.Log(fmt.Sprintf("Trying to sync git changes to cluster, from commit '%s' to commit '%s'", c.oldTagRev, c.newTagRev))
```

Comment:
I missed the fact that you did not include a key for the message, I think you have two options:

```suggestion
	d.Logger.Log("info", fmt.Sprintf("trying to sync git changes to the cluster, from commit '%s' to commit '%s'", c.oldTagRev, c.newTagRev))
```

or

```suggestion
	d.Logger.Log("info", "trying to sync git changes to the cluster", "old", c.oldTagRev, "new", c.neTagRev)
```

The latter matches https://github.com/fluxcd/flux/blob/4ed1b29667bbb0a82872c7201ce965016e3c68a7/pkg/daemon/loop.go#L220

Reply:
Thanks, the latter makes more sense. Just need to fix a typo (c.newTagRev vs c.neTagRev) and pushing.

Reply:
Can you please smash your commits together and rebase on `master` too?